src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsAmazonObject.java

```diff
@@ -66,10 +87,20 @@ public void verifyGPTParams() {
         }
     }
 
-    public AdsAmazonObject clickAmazonArticleLink() {
+    public AdsAmazonObject clickAmazonArticleLink(String linkSelectoryInCss) {
         WebElement amazonArticleLink = driver.findElement(By.cssSelector(AMAZON_ARTICLE_LINK_CSS));
         waitForElementByElement(amazonArticleLink);
         amazonArticleLink.click();
         return this;
     }
+
+    public String getAmazonLinkCssSelector(String linkName) {
+        if (linkName.equals("AmazonArticle")) {
+            return AMAZON_ARTICLE_LINK_CSS;
+        } else if (linkName.equals("AmazonSecondArticle")) {
```

Comment:
I would think about using `ImmutableMap` here if we are going add new linkNames in the future.